recipe/build-libadios.sh

```diff
@@ -65,36 +54,16 @@ cmake               \
     -DADIOS2_USE_ZeroMQ=ON                    \
     -DADIOS2_USE_ZFP=ON                       \
     -DBUILD_SHARED_LIBS=ON                    \
-    -DBUILD_TESTING=${RUN_TESTS}              \
+    -DBUILD_TESTING=OFF                       \
     -DCMAKE_BUILD_TYPE=Release                \
     -DCMAKE_INSTALL_LIBDIR=lib                \
     -DCMAKE_INSTALL_PREFIX="${PREFIX}"        \
     -DPython_EXECUTABLE:FILEPATH="${PYTHON}"  \
     -DPython_INCLUDE_DIR="$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])")" \
     -DPNG_PNG_INCLUDE_DIR="${PREFIX}"
 
-cmake --build build "-j${CPU_COUNT}" -v
-
-if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" && "${RUN_TESTS}" == "ON" ]]
-then
-    # SST: Flaky tests
-    exclude_tests="SST"
-    # Disable the Makefile install test since it relies on the adios2-config
-    # script which is not being build anyways since it is being flaky in
-    # several configurations.
-    exclude_tests+="|Install.Make.*"
-    if [[ "${target_platform}" =~ osx ]]
-    then
-        exclude_tests+="|Test.Engine.DataMan1D.Serial"
-        exclude_tests+="|Test.Engine.DataMan1xN.Serial"
-        exclude_tests+="|Test.Engine.DataManSingleValues"
-        exclude_tests+="|Remote.BPWriteReadADIOS2stdio.GetRemote"
-        exclude_tests+="|Remote.BPWriteMemorySelectionRead.GetRemote"
-        exclude_tests+="|Remote.BPWriteMemorySelectionRead.FileRemote"
-    fi
+cmake --build build "-j$((CPU_COUNT*2))" -v
```

Comment:
compilation process are also IO bounded.

Reply:
I would not exceed this. Some (most) IO services check process utilization and oversubscription can lead to processes being killed sporadically.